### ISSUE TITLE
keras 1.1.0 conv problems?

### ISSUE THREAD

**Issue by andreisun** [fchollet/keras#3956]
Hi!
The model  containing various Convolution2D, MaxPooling2D trained to logloss .2 on the Keras **1.0.6** on the **1.1.0** "hanging" on .69 and changing LR or other parameters - no improving.
Only trimming conv layers I getting some usable results. 
The same problem on the kaggle docker with 1.1.0 keras: [https://www.kaggle.com/andreisun/dogs-vs-cats-redux-kernels-edition/cat-nn-126](url)
Local machine - Win10 + GeForce GT 540M
Google group: [https://groups.google.com/forum/#!topic/keras-users/igc97jbLmn4](url)

**Comment by zolia** [comment]
I can confirm this too. CNN is not converging with cifir-10 example in Keras 1.1.0
Training on CPU.

Using TensorFlow backend.
X_train shape: (50000, 32, 32, 3)
50000 train samples
10000 test samples
Using real-time data augmentation.
Epoch 1/200
50000/50000 [==============================] - 352s - loss: 2.3036 - acc: 0.1019 - val_loss: 2.3031 - val_acc: 0.1000
Epoch 2/200
50000/50000 [==============================] - 363s - loss: 2.3035 - acc: 0.0998 - val_loss: 2.3030 - val_acc: 0.1000
Epoch 3/200
50000/50000 [==============================] - 381s - loss: 2.3036 - acc: 0.0980 - val_loss: 2.3034 - val_acc: 0.1000
Epoch 4/200
50000/50000 [==============================] - 385s - loss: 2.3033 - acc: 0.0997 - val_loss: 2.3030 - val_acc: 0.1000
Epoch 5/200
...

After removing two convolution layers it started to train: 

Using TensorFlow backend.
X_train shape: (50000, 32, 32, 3)
50000 train samples
10000 test samples
Using real-time data augmentation.
Epoch 1/200
50000/50000 [==============================] - 269s - loss: 1.8371 - acc: 0.3304 - val_loss: 1.4781 - val_acc: 0.4588
Epoch 2/200
50000/50000 [==============================] - 267s - loss: 1.4844 - acc: 0.4659 - val_loss: 1.3061 - val_acc: 0.5272
Epoch 3/200
50000/50000 [==============================] - 262s - loss: 1.3432 - acc: 0.5159 - val_loss: 1.1288 - val_acc: 0.5997
Epoch 4/200
50000/50000 [==============================] - 260s - loss: 1.2446 - acc: 0.5549 - val_loss: 1.0532 - val_acc: 0.6226
Epoch 5/200
50000/50000 [==============================] - 259s - loss: 1.1807 - acc: 0.5771 - val_loss: 0.9740 - val_acc: 0.6574

Antanas
